lua/scnvim/help.lua

```diff
@@ -67,4 +68,78 @@ function M.handle_method(name, target_dir)
   end
 end
 
+function M.render_all(include_extensions, callback)
+  include_extensions = include_extensions or true
+  sclang = sclang or require'scnvim.sclang'
+  local settings = vim.fn['scnvim#util#get_user_settings']()
+  local render_args = vim.fn['scnvim#util#get_scdoc_render_args']()
+  local render_prg = settings.paths.scdoc_render_prg
+  local cmd = string.format('SCNvimDoc.renderAll(%s)', include_extensions)
+  sclang.eval(cmd, function()
+    sclang.eval('SCDoc.helpTargetDir', function(help_path)
+      local sep = utils.path_sep
+      local sc_help_dir = help_path .. sep .. 'Classes'
```

Comment:
Should this directory maybe be available outside of this? Might be useful for hackers :)

Reply:
Something in the spirit of the `get_snippets()` method maybe? Like `get_sc_help_dir()` as a method that is exported in the utils module or something

Reply:
It would still require a callback to get the return value and that `sclang` is running, so I would prefer `sclang.eval` as it makes it more transparent as to why that is. 

With that said, I'm working on a `path` module in the lua-config branch, maybe it would make sense to "cache" some of these paths like this one but maybe others like `userAppSupportDir` etc.?